src/strings/PowerFxResources.en-US.resx

```diff
@@ -4464,4 +4464,23 @@
       A partial operator is the 2nd part of a statement `[Partial Op]` and can be one of "And", "Or", "Table" or "Record".
       It's used to determine how to combine multiple expressions with the same name and operator.</comment>
   </data>
+  <data name="ErrNeedTable" xml:space="preserve">
+    <value>Argument needs to be a table.</value>
+  </data>
+  <data name="AboutTableConcatenate" xml:space="preserve">
+    <value>Creates a new table by combining the all the rows from the specified input tables.</value>
+    <comment>Description of 'TableConcatenate' function.</comment>
+  </data>
+  <data name="TableConcatenateArg1" xml:space="preserve">
+    <value>table</value>
+    <comment>function_parameter - Argument of the TableConcatenate function - A table whose rows will be appended to the resulting table.</comment>
+  </data>
+  <data name="AboutTableConcatenate_table" xml:space="preserve">
+    <value>A table whose rows will be appended to the resulting table.</value>
```

Comment:
>value [](http://example.com/codeflow?start=5&length=5)

Every new resource must have a comment explaining what the term means and the context in which the string is shown to the user. That prevents translation issues that we had in the past.